Airports/EGPH/SMR/Labels.txt

```diff
@@ -58,10 +58,18 @@
 "9" N055.56.57.290 W003.21.51.101 standHold
```

Comment:
Although not very important, stands 2A, 3A, 4A, 9A, 10A have been added.
![default](https://user-images.githubusercontent.com/18355854/29998203-c56cdaf6-902d-11e7-8dc8-3b71880842de.png)

Reply:
These stands have been there ages in reality. I haven't added them as they won;t show nicely, I didn't think it necessary to add these lines.